src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -94,6 +94,7 @@ namespace cryptonote
   //-----------------------------------------------------------------------------------------------
   bool core::update_checkpoints()
   {
+    return true;
```

Comment:
I think I slipped that in for debugging because my DNS settings were being annoying and accidentally committed it, thanks for noticing!